packages/api-clients/open-api/catalogApi.yml

```
@@ -2042,8 +2113,8 @@ components:
           type: array
           items:
             type: string
-        templateVersionRef:
-          $ref: "#/components/schemas/EServiceTemplateVersionRef"
+        interfaceTemplateMetadata:
+          $ref: "#/components/schemas/TemplateInstanceInterfaceMetadata"
```

I think this change is unrelated/unwanted
```suggestion
        templateVersionRef:
          $ref: "#/components/schemas/EServiceTemplateVersionRef"
```